corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -576,7 +580,7 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
     Form.prototype.constructor = Container;
 
     /**
-     * Represents a group of GroupedQuestionTileRow which contains questions.
+     * Represents a group of GroupedElementTileRow which contains questions.
```

Comment:
nit: I think it should be "which contains Questions or Groups". The different uses of the word "group" is getting confusing, maybe using capital "Group" to represent the object would help.

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/22c152e7d701ff9c3c4aa94c0a58a62154771463